### PR TITLE
fix(size): change density to size

### PR DESCRIPTION
**Pull request by rachelbt**
First stem in moving to using sizing tokens

### CODE REVIEW

**Comment by yinonov** [comment]
add this to the list of removals (enlarged number field)

image

File: change/@vonage-vivid-40f0613f-f6a4-4d06-b516-92c0b52da217.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "change densety to size and extended to expanded",
```
Comment (yinonov) [inline change/@vonage-vivid-40f0613f-f6a4-4d06-b516-92c0b52da217.json:3]:
```suggestion
  "comment": "change density to size and extended to expanded",
```

**Review by yinonov — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/enums.ts [hunk 2]
```diff
@@ -30,19 +30,34 @@ export enum Appearance {
 	Subtle = 'subtle',
 	Ghost = 'ghost',
 }
-
+//need to be removed after remove in text-field
 export enum Density {
```
Comment (yinonov) [inline libs/components/src/lib/enums.ts:34]: are we still using it?

File: libs/components/src/lib/icon/README.md [hunk 3]
```diff
@@ -50,13 +50,13 @@ While `type` fetches an icon from the Vivid predefined set of icons, `svg` prope
 
 Use the `size` attribute to apply a predefined size to the icon.
 
-- Type: `'small'` | `'medium'` | `'large'`
-- Default: `'medium'`
+- Type: `'condensed'` | `'normal'` | `'expanded'`
```
Comment (yinonov) [inline libs/components/src/lib/icon/README.md:53]: Icon should be the same as progress ring. next PR?

**Review by yinonov — COMMENTED** [review]
few small comments

File: libs/components/src/lib/icon/README.md (hunk 3 above)
Comment (rachelbt) [reply libs/components/src/lib/icon/README.md:53]: Not sure it does. either way - that sounds like a different PR

File: libs/components/src/lib/enums.ts (hunk 2 above)
Comment (rachelbt) [reply libs/components/src/lib/enums.ts:34]: no. you re right - we removed the density there. 
I'll remove it

Comment (rachelbt) [reply libs/components/src/lib/enums.ts]: removed

**Review by yinonov — APPROVED** [review]

**Pull request merged**